dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/service/impl/UsersServiceImpl.java

```diff
@@ -155,7 +157,7 @@ public Map<String, Object> createUser(User loginUser,
             putMsg(result, Status.REQUEST_PARAMS_NOT_VALID_ERROR, msg);
             return result;
         }
-        if (!isAdmin(loginUser)) {
+        if (!canOperatorPermissions(loginUser,null, AuthorizationType.USER, FuncPermissionEnum.CREATE_USER.toString())) {
             putMsg(result, Status.USER_NO_OPERATION_PERM);
```

Comment:
for 'User Api', the commercial version rewrites the entire user module, so the next step in this block is to use whether the Api is disabled instead of whether it has permission.